### PR TITLE
Replace http with https in static URLs

### PR DESCRIPTION
**Pull request by vvasiloi**
| Q               | A
| --------------- | -----
| Branch?         | 1.12
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | replaces https://github.com/Sylius/Sylius/pull/11922
| License         | MIT

### CODE REVIEW

File: adr/2021_02_05_unified_api_prefix.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ header in addition to this path, however it may be misleading for consumers.
 - https://github.com/api-platform/core/issues/972
 - https://stackoverflow.com/questions/389169/best-practices-for-api-versioning
 - https://www.troyhunt.com/your-api-versioning-is-wrong-which-is/
-- http://codebetter.com/howarddierking/2012/11/09/versioning-restful-services/
+- https://codebetter.com/howarddierking/2012/11/09/versioning-restful-services/
```
Comment (vvasiloi) [inline adr/2021_02_05_unified_api_prefix.md:59]: This link looks dead on both http and https.

**Comment by vvasiloi** [comment]
Currently https://api.sylius.com/ redirects to https://docs.sylius.com/en/latest/, but http://api.sylius.com/ works.

File: adr/2021_02_05_unified_api_prefix.md (hunk 1 above)
Comment (lchrusciel) [reply adr/2021_02_05_unified_api_prefix.md:59]: lets remove it then

**Comment by lchrusciel** [comment]
website, that you mentioned will be put down at some moment. For now, lets remove it from the docs

**Comment by vvasiloi** [comment]
> Currently https://api.sylius.com/ redirects to https://docs.sylius.com/en/latest/, but http://api.sylius.com/ works.

@lchrusciel what about this one?

**Comment by vvasiloi** [comment]
http://api.sylius.com/ was shut down and all references to it were removed.

**Review by Zales0123 — COMMENTED** [review]
👍 but shouldn't we base it on 1.10? It's definitely to mention https instead of http in ever docs we support 🖖

**Comment by vvasiloi** [comment]
> but shouldn't we base it on 1.10?

Sure, why not. It's only static files.

**Comment by jakubtobiasz** [comment]
Before anyone start yelling at me for "Merge branch '1.12' into https-static" commit:
Locally I couldn't manage to get this conflict, lol. I had to use web to resolve two conflicts, and that's why this commit happened :D.

**Review by jakubtobiasz — APPROVED** [review]

**Comment by jakubtobiasz** [comment]
Thank you, @vvasiloi!

**Pull request merged**